modules/apps/object/object-test/src/testFunctional/tests/ObjectAutoGeneratedView.testcase

```diff
@@ -23,48 +23,162 @@ definition {
+
+		ObjectAdmin.goToCustomObject(objectName = "CustomObject");
+
+		ObjectPortlet.viewEntry(entry = "By building a vibrant business, making technology useful, and investing in communities, we make it possible for people to reach their full potential to serve others.");
 	}
 
 	@description = "LPS-142659 - Verify it is possible to edit a entry of Clob field and view the update on entry table"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanEditClobField {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Custom Field",
+			fieldName = "customObjectField",
+			fieldType = "Clob",
+			isRequired = "false",
+			objectName = "CustomObject");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+
+		ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+			fieldName = "customObjectField",
+			objectName = "CustomObject",
+			value = "We make it possible for people to reach their full potential to serve others.");
+
+		ObjectAdmin.goToCustomObject(objectName = "CustomObject");
 
-		// TODO LPS-145702 CanEditClobField pending implementation
+		ObjectPortlet.viewEntry(entry = "We make it possible for people to reach their full potential to serve others.");
 
+		ObjectPortlet.viewClobEntryDetails(clobEntry = "We make it possible for people to reach their full potential to serve others.");
+
+		ObjectAdmin.editClobEntryName(clobEntry = "All people are inherently valuable; who we are is as important as what we can do.");
+
+		ObjectAdmin.goToCustomObject(objectName = "CustomObject");
+
+		ObjectPortlet.viewEntry(entry = "All people are inherently valuable; who we are is as important as what we can do.");
 	}
 
 	@description = "LPS-142659 - Verify that a Clob entry is correctly displayed on the auto-generated table on the Custom Object Portlet"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test ClobDisplayed {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Custom Field",
+			fieldName = "customObjectField",
+			fieldType = "Clob",
+			isRequired = "false",
+			objectName = "CustomObject");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+
+		ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+			fieldName = "customObjectField",
+			objectName = "CustomObject",
+			value = "We make it possible for people to reach their full potential to serve others.");
 
-		// TODO LPS-145701 ClobDisplayed pending implementation
+		ObjectAdmin.goToCustomObject(objectName = "CustomObject");
 
+		ObjectPortlet.viewClobEntryDetails(clobEntry = "We make it possible for people to reach their full potential to serve others.");
+
+		ObjectAdmin.assertClobTypeText(clobType = "We make it possible for people to reach their full potential to serve others.");
```

Comment:
Here you need to use the `ObjectPortlet.viewEntry` macro (just like the `PicklistDisplayed` test).